src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -71,12 +71,14 @@ def _get_resource_group(self, **kwargs):
 
 class ServerMgmtScenarioTest(ScenarioTest):
 
+    @live_only()
```

Comment:
@nasc17 Please be aware of this change. For more context, please refer to the PR description